content/2020/2020-09/2020-09-25-pulsar-weekly.md

```diff
@@ -0,0 +1,133 @@
+
+    https://github.com/apache/pulsar/pull/8127(@[srkukarni](https://github.com/srkukarni))
+
+- [Websocket] The WebSocket API supports unsubscribing from a topic.
+
+    https://github.com/apache/pulsar/pull/8068(@[315157973](https://github.com/315157973))
+
+- [Websocket] The WebSocket interface supports decoding URL for the topic name and subscription.
+
+    https://github.com/apache/pulsar/pull/8072(@[315157973](https://github.com/315157973))
+
+- [Broker] Support configuring the number of channels per bookie.
+
+    https://github.com/apache/pulsar/pull/7910(@[rdhabalia](https://github.com/rdhabalia))
+
+- [Functions] Support specifying the environment based on the secrets provider for source/sink/functions localrunner.
+
+    https://github.com/apache/pulsar/pull/8098(@[srkukarni](https://github.com/srkukarni))
+
+- [Client] Add a REST endpoint to trigger `backlogQuotaCheck`.
+
+    https://github.com/apache/pulsar/pull/8045(@[315157973](https://github.com/315157973))
+
+- [pulsar-client] Support non-durable subscription for pulsar-client commands.
+
+    https://github.com/apache/pulsar/pull/8100(@[315157973](https://github.com/315157973))
+
+- [Functions] Enable reporting functions worker JVM metrics through Prometheus.
+
+    https://github.com/apache/pulsar/pull/8097(@[jerrypeng](https://github.com/jerrypeng))
+
+- [Broker] Add replicated subscription check to `checkInactiveSubscriptions`.
+
+    https://github.com/apache/pulsar/pull/8066(@[shustsud](https://github.com/shustsud))
+
+- [Client] Support forcedly deleting all resources related to the namespace.
+
+    https://github.com/apache/pulsar/pull/7993(@[murong00](https://github.com/murong00))
+
+- [Topic Policy] Support setting, getting, and removing the subscription dispatch rate on the topic level.
+
+    https://github.com/apache/pulsar/pull/8087(@[hangc0276](https://github.com/hangc0276))
+
+- [Topic Policy] Support setting, getting, and removing the subscribe rate on the topic level.
+
+    https://github.com/apache/pulsar/pull/7991(@[hangc0276](https://github.com/hangc0276))
+
+- [Client] Support dynamically loading the Truststore certificate through the  input stream.
+	
+    https://github.com/apache/pulsar/pull/7442(@[rdhabalia](https://github.com/rdhabalia))
+
+- [Broker] Replace `IdentityHashMap` with `set -- Collections.newSetFromMap(new ConcurrentHashMap<>())` to hold the reference of producers/consumers .
+
+    https://github.com/apache/pulsar/pull/8051(@[Technoboy-](https://github.com/Technoboy-))
+
+### Notable Bug Fix
+
+- [Functions] Fix Pulsar Functions threaded integration tests.
+
+   https://github.com/apache/pulsar/pull/8118(@[jerrypeng](https://github.com/jerrypeng))
+
+- [Client] Use the bytes of messages instead the number of messages for the producer to pend messages.
+
+    https://github.com/apache/pulsar/pull/7957(@[Renkai](https://github.com/Renkai))
+
+- [Client] Fix client lookup hangs when the broker restarts.
```

Comment:
replace "hang" with another word

Reply:
Updated as : Fix the issue that the the client lookup hangs when the broker restarts.